src/arrayMethodReduce.js

```diff
@@ -6,6 +6,19 @@
 function applyCustomReduce() {
   [].__proto__.reduce2 = function(callback, initialValue) {
     // write code here
+    let y = 0;
```

Comment:
```suggestion
    let startIndex = 0;
```
давай змінним зрозумілі імена